cadc-vos/src/main/java/ca/nrc/cadc/vos/client/ClientTransfer.java

```diff
@@ -545,41 +537,33 @@ private void doServerTransfer()
         {
             URL phaseURL = new URL(jobURL.toExternalForm() + "/phase");
             String parameters = "PHASE=RUN";
-            
-            HttpURLConnection connection = (HttpURLConnection) phaseURL.openConnection();
-            if (connection instanceof HttpsURLConnection)
-                initHTTPS((HttpsURLConnection) connection);
-            connection.setRequestMethod("POST");
-            connection.setRequestProperty("Content-Length", "" + Integer.toString(parameters.getBytes().length));
-            connection.setRequestProperty("Content-Type", "application/x-www-form-urlencoded");
-            connection.setInstanceFollowRedirects(false);
-            connection.setUseCaches(false);
-            connection.setDoOutput(true);
-            connection.setDoInput(true);
-
-            OutputStream outputStream = connection.getOutputStream();
-            outputStream.write(parameters.getBytes("UTF-8"));
-            outputStream.close();
-
-            int responseCode = connection.getResponseCode();
-            String responseMessage = connection.getResponseMessage();
-            String errorBody = NetUtil.getErrorBody(connection);
-            if (StringUtil.hasText(errorBody))
-                responseMessage += ": " + errorBody;
-            
-            switch (responseCode)
+
+            HttpPost transfer = new HttpPost(phaseURL, parameters.toString(),
+                    "application/xml", false);
```

Comment:
The constructor public HttpPost(URL url, Map<String, Object> map, boolean followRedirects) should be used here instead:
  - the parameters should go in the map
  - since there isn't an XML document being posted there's no need to set the content-type.